java/src/test/java/com/adhoc/flight/TestAdhocFlightClient.java

```diff
@@ -76,7 +88,11 @@ public class TestAdhocFlightClient {
     }
   };
 
+  @InjectMocks
   private AdhocFlightClient client;
```

Comment:
Do we really want a mock here or an actual client and mock the server ?

Reply:
I agree that it's better to mock the server than a client.

Reply:
As we discussed in the private message, we want to test AdhocFlightClient here, so it's ok to mock FlightClient and check that we are using it as expected.

Reply:
I changed this test to more expclicit one in another test file